Automattic/wp-calypso
# Marketplace: Add empty page for plans

#### Proposed Changes

* Adds a new /plugins/plans/:site route with empty content.

#### Testing Instructions

* Visit http://calypso.localhost:3000/plugins/plans/testing705633960.wordpress\com with your own test site in the url
* View the new (empty) plans page:
![Screenshot 2022-09-13 at 15-18-46 Plugins Plan Upgrade ‹ Business - Simple Site — WordPress com](https://user-images.githubusercontent.com/811776/189815372-ec3fc221-cd44-4a83-b2e0-a6ada921f1b7.png)
* Confirm plugin details pages still load as this shares a similar route pattern

Related to #67574
